pkg/kn/commands/describe.go

```diff
@@ -180,6 +180,32 @@ func WriteConditions(dw printers.PrefixWriter, conditions []apis.Condition, prin
 	}
 }
 
+// Writer a slice compact (printDetails == false) in one line, or over multiple line
+// with key-value line-by-line (printDetails == true)
+func WriteSliceDesc(dw printers.PrefixWriter, s []string, label string, printDetails bool) {
```

Comment:
Before we forget, should we then also remove https://github.com/knative/client/blob/aafdb9a2387b5a50d880a2ee563fcf2786217fc8/pkg/kn/commands/service/describe.go#L325-L345 and let `kn service describe` use this method ?

Reply:
Done